manifold/src/impl.cpp

```diff
@@ -174,6 +173,7 @@ struct InitializeBaryRef {
     int tri = thrust::get<1>(inOut);
 
     // Leave existing meshID if input is negative
+    // TODO: Why do we need this?
```

Comment:
Yeah, not super clear. It makes it so when I make a `sphere` and call `ReinitializeReference` without an argument, it doesn't bother renumbering. I can't remember a reason that's important. If you'd like to simplify it and nothing breaks, be my guest.